Makefile

```
@@ -452,6 +452,28 @@ tag_push:
+# `production`. Run from the commit whose pubspec carries the version to
+# promote, because the workflow refuses a tag that disagrees with it.
+.PHONY: play_tag_push
+play_tag_push:
+	@test -n "$(LOTTI_VERSION)" || { echo "LOTTI_VERSION is empty: install yq (brew install yq)" >&2; exit 1; }
+	@test -n "$(PLAY_TRACK)" || { echo "PLAY_TRACK is unset: use android_closed_testing or android_release" >&2; exit 1; }
+	git tag play/$(PLAY_TRACK)/$(LOTTI_VERSION)
+	git push origin play/$(PLAY_TRACK)/$(LOTTI_VERSION)
+
+.PHONY: android_closed_testing
+android_closed_testing: PLAY_TRACK = alpha
+android_closed_testing: play_tag_push
```

checkmake (0.3.2)

[warning] Target "android_closed_testing" defined multiple times (lines 470 and 471).

(uniquetargets)